auth/jwt_controller.go

```diff
@@ -21,18 +21,39 @@ import (
 // the "auth.jwt.secret" config entry.
 // The token is set to expire in the amount of seconds defined by
 // the "auth.jwt.expiry" config entry.
-func GenerateToken(id interface{}) (string, error) {
+func GenerateToken(username interface{}) (string, error) {
```

Comment:
I changed the parameter name here, because what we really expect is `username` field of user model. Not `id`. It was confusing IMHO.

Reply:
Good initiative